lib/rules/no-trailing-spaces.js

```diff
@@ -34,7 +40,7 @@ module.exports = {
     create(context) {
         const sourceCode = context.getSourceCode();
 
-        const BLANK_CLASS = "[ \t\u00a0\u2000-\u200b\u2028\u2029\u3000]",
+        const BLANK_CLASS = "[ \t\u00a0\u2000-\u200b\u3000]",
```

Comment:
This looks correct, but I'm not sure why it originally included line breaks? Maybe there was a reason for it?